gnmidiff/gostruct_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gnmidiff
+
+import "github.com/openconfig/ygot/ygot"
+
+// Device represents the /device YANG schema element.
```

Comment:
Given that this verbatim reproduces part of the schema, consider using `exampleoc`?